Automattic/pocket-casts-ios
# Move WatchOS project files to single folder/group

Fixes #

This is a follow up o[n the upgrading of the WatchOS project to be a single target](https://github.com/Automattic/pocket-casts-ios/pull/1505).
On this PR I moved the files that were part of the extension inside the main app folder.
I also removed the duplicates image assets that were shared between projects and merged together the assets files in a single file.

## To test

- Start the Watch OS app on a device
- Smoke test the app to see if all assets are showing correctly.

## Checklist
